Assets/Scripts/Weapons/Projectile.cs

```diff
@@ -1,32 +1,34 @@
 using Mirror;
 using UnityEngine;
 using UnityEngine.Tilemaps;
-using UnityEngine.UIElements;
+
 
 public class Projectile : NetworkBehaviour
 {
-    public float speed = 10f;
     public float lifetime = 15f;
-    public int damage = 5;
 
     [SyncVar]
     private Vector2 direction;
 
     private float timeAlive = 0f;
+	private float speed = 10f;
+	private int damage = 5;
 
-    private Tilemap tilemap;
+	private Tilemap tilemap;
```

Comment:
```suggestion
    private Tilemap tilemap;
```
używajmy spacji zamiast tabów żeby było jednolicie

Reply:
Done